docker/bin/run_integration_tests.sh

```diff
@@ -25,6 +25,10 @@ case $1 in
     PLATFORM="Windows 7"
     MARK_EXPRESSION="sanity and not headless"
     ;;
+  headlessplus)
```

Comment:
Could we use a more descriptive name here perhaps? We could even just call this `download`, and then explicitly include both `headless` and `download` in the YAML files where appropriate?

Reply:
I wrestled with that too. I don't know what to call it. Could just have both and add the download tests to those places... I guess that does make most sense.